### ISSUE TITLE
Calendar: today highlight with multiple cal pages

### ISSUE THREAD

**Issue by rshadow** [copycat-killer/lain#170]
Reproduce:
* Use calendar with additional parameters: 
```lua
lain.widgets.calendar:attach(widgetdate, {
    font = "Monospace", position = "top_right", post_cal = "-A1"
})
```
* On mouse over `widgetdate` get calendar with two months. In first and second month same date is highlighted.

![2016-02-27 16 44 54](https://cloud.githubusercontent.com/assets/737152/13373199/c300011e-dd71-11e5-99bc-df37bdd5cf69.png)

**Comment by rshadow** [comment]
![2016-02-27 16 44 54](https://cloud.githubusercontent.com/assets/737152/13373199/c300011e-dd71-11e5-99bc-df37bdd5cf69.png)

**Comment by copycat-killer** [comment]
Why did you open 2 more thread? One was enough to contain all.

The widget wasn't meant to be used with more than one calendar page, the `post_cal` argument is there only to solve issues related to `cal` implementation on different distros. Tell me if f7601c7 solves it.

The problem referred int #171 it's due to `cal` probably always coloring the output. Set `post_cal` like this:

```lua
lain.widgets.calendar:attach(widgetdate, {
    font = "Monospace", position = "top_right", post_cal = "-A1 --color=never"
})
```

As for writing a month under the current date, it's impossible since the big number is an icon. Besides, it's redundant with today highlighting. Also, why do you want that in the notification, when you can have it in the widget?

My calendar is supposed to be simple. If you really want such a feature, you might want to take a look at [blingbling](https://github.com/cedlemo/blingbling).

**Comment by rshadow** [comment]
It is a habit: one bug one ticket.

Thank you for your time.

**Comment by rshadow** [comment]
![2016-03-01 17 13 33](https://cloud.githubusercontent.com/assets/737152/13429460/7626fcf0-dfd1-11e5-8c57-e6c19a725660.png)

Today I have this bug

**Comment by copycat-killer** [comment]
Recursively closed by #174.
